lib/Zonemaster/Backend/Config.pm

```diff
@@ -184,6 +185,11 @@ sub parse {
         $obj->_set_SQLITE_database_file( $value )
           if $obj->DB_engine eq 'SQLite';
     }
+    if ( defined( my $value = $ini->val( 'LANGUAGE', 'locale' ) ) ) {
+        if ( $value eq "" ) {
+            push @warnings, "Use of empty LANGUAGE.locale propery is deprecated. Remove LANGUAGE.locale entry or specify LANUGAGE.locale = en_US instead.";
```

Comment:
The documentation says: "Leaving the locale key empty or absent is deprecated. Always configure it with supported locale tags." Having no LANGUAGE.locale is deprecated.

Reply:
I removed the deprecation of absent LANGAUGE.locale. That way we don't have to force users who don't care about native language support to add this property to their configuration files when we finally remove the feature. Having a default value of "en_US" is perfectly reasonable.